### PR TITLE
expenses openapi spec

### PR DESCRIPTION
**Pull request by jayachandhar**

### CODE REVIEW

File: src/components/schemas/fields.yaml [hunk 1]
```diff
@@ -170,11 +170,60 @@ currency:
...
+
+foreign_currency:
+  type: string
+  description: >
+    The three-letter currency code following ISO standard <a
+    href='https://www.iso.org/iso-4217-currency-codes.html'>ISO 4217</a>.
+
+    Original currency signifies the currency in which the object
+    (expense/advance/refund/etc.) was originally created.
+  example: INR
```
Comment (snarayanank2) [inline src/components/schemas/fields.yaml:190]: change INR in examples to GBP or something

File: src/components/schemas/file.yaml [hunk 2]
```diff
@@ -56,10 +57,14 @@ file_out_embed:
       type: string
       enum:
         - INTEGRATION
+        - RECEIPT
```
Comment (snarayanank2) [inline src/components/schemas/file.yaml:60]: don't we need to add thumbnail and other types we support as well?

**Review by snarayanank2 — CHANGES_REQUESTED** [review]

File: src/components/schemas/expense.yaml [hunk 3]
```diff
@@ -2,406 +2,404 @@ expense_out:
...
-    #   description: |
-    #     This represents the id of employee who updated this expense.
-    #   example: ouGgsdfdq
-    # last_updater:
-    #   readOnly: true
-    #   allOf:
-    #     - $ref: ./employee.yaml
-    #   description: |
-    #     Details of employee who updated this expense.
+fyler_expense_in:
```
Comment (aditya999123) [inline src/components/schemas/expense.yaml:387]: nice

File: src/components/schemas/file.yaml (hunk 2 above)
Comment (jayachandhar) [reply src/components/schemas/file.yaml:62]: other types are thumbnails and exports, 
thumbnails implementation can change, can be added when we are doing frontend changes.
exports will be required only for csv/excel or pdf exports, can be added when we are supporting exports in platform side.

**Pull request merged**